backend/controllers/cellMove/cellMoveHistory.js

```diff
@@ -0,0 +1,137 @@
+const moment = require('moment')
+
+const { formatName, formatLocation, stripAgencyPrefix, putLastNameFirst } = require('../../utils.js')
+
+const latestBedAssignment = (left, right) => right.bedAssignmentHistorySequence - left.bedAssignmentHistorySequence
+const formatLocationDescription = (description, agencyId) => formatLocation(stripAgencyPrefix(description, agencyId))
+
+const matchesLocationPrefix = (description, locationPrefix) => {
+  if (locationPrefix.length > description.length) return false
+
+  const data = description.substr(0, locationPrefix.length)
+  return data === locationPrefix
+}
+
+const dateTimeFormat = 'YYYY-MM-DDTHH:mm'
+
+const sortByMostEarliestFirst = (left, right) => {
+  const leftDate = moment(left.assignmentDateTime, dateTimeFormat)
+  const rightDate = moment(right.assignmentDateTime, dateTimeFormat)
+
+  if (leftDate.isAfter(rightDate, 'minute')) return 1
+  if (leftDate.isBefore(rightDate, 'minute')) return -1
+
+  return 0
+}
+
+const prisonApiLocationDescription = async (res, whereaboutsApi, locationKey, userCaseLoad) => {
```

Comment:
Duplicate code, we need to pull this out into a common module. One for another PR.